### ISSUE TITLE
Quotes formatting breaks valid config

### ISSUE THREAD

**Issue by SkeLLLa** [raynigon/vscode-nginx-formatter#9]
Here's an example of valid nginx.conf.

```
http {
    include /etc/nginx/mime.types;
    default_type application/octet-stream;
    access_log off;
    log_format json_combined escape=json
      '{'
        '"timeLocal":"$time_local",'
        '"remoteAddr":"$remote_addr",'
        '"forwarderFor":"$http_x_forwarded_for",'
        '"remoteUser":"$remote_user",'
        '"request":"$request",'
        '"status": "$status",'
        '"bytesSent":"$bytes_sent",'
        '"requestTime":"$request_time",'
        '"httpReferrer":"$http_referer",'
        '"gzipRatio":"$gzip_ratio",'
        '"httpUserAgent":"$http_user_agent"'
      '}';
    tcp_nopush on;
    tcp_nodelay on;
}
```

Formatter breaks it this way:
```
http {
  include /etc/nginx/mime.types;
  default_type application/octet-stream;
  access_log off;
  log_format json_combined escape=json
  ' {
    '
    '"timeLocal":"$time_local",'
    '"remoteAddr":"$remote_addr",'
    '"forwarderFor":"$http_x_forwarded_for",'
    '"remoteUser":"$remote_user",'
    '"request":"$request",'
    '"status": "$status",'
    '"bytesSent":"$bytes_sent",'
    '"requestTime":"$request_time",'
    '"httpReferrer":"$http_referer",'
    '"gzipRatio":"$gzip_ratio",'
    '"httpUserAgent":"$http_user_agent"'

  }
  ';
  tcp_nopush on;
  tcp_nodelay on;
}
```

Probably it was caused by some new changes, cause ~ 1 year ago it was working fine with such file.

Looks like nginxbeautifier bug.

**Comment by raynigon** [comment]
Thanks for writing this issue.
See https://github.com/vasilevich/nginxbeautifier/issues/15 for further information.
I will release a fix as soon as possible.
The formatter was changed about six months ago.

**Comment by raynigon** [comment]
Released with v0.0.11
